content/guides/user-guides/multi-cloud/storage-service-plan/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+
+## Introduction to Storage Service Plan(SSP) management
+SODA is aimed at addressing the storage integration challenges of both the Cloud Native environment and traditional IT environment. SODA multi-cloud allows service providers(admin) or users to manage distributed cloud and on-premise environments from a single platform.
+
+
+SSP management allows service providers to abstract actual cloud or on-premise environments(backends) from their users.
+In other words, SSP is a policy for the users. A service provider can decide and create different types of SSP policy for different tenants of users.
+
+## Getting Started!
+This is a guide that shows how to install, configure, and use SSP management features in a simple SODA setup.
+By following this guide you will be able to install SODA multi-cloud with creating backends and buckets, uploading and archiving objects.
+
+## Installing SODA multi-cloud
+
+Follow the installation guide for multi-cloud installation.
+
+- [SODA Local Cluster Installation through Ansible On Ubuntu](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) (Recommended)
+
+
+* To enable SSP feature, and experience through SODA Dashboard modify the "enable_storage_service_plans" variable and then install:
+
+  group_vars/common.yml
+
+  ```
+  enable_storage_service_plans: true
+  ```
+
+## Open SODA User Interface Dashboard
+
+SODA UI can be accessed via
+
+`http://{your_host_ip}:8088/#/home`
+
+login  using the default admin credentials: `admin/opensds@123`.
+
+![multi-cloud UI image  ](soda_login_page.png?raw=true)
+
+### Create new backend as cloud or on-prem storage: [Admin Operation]
+
+**Note**: When SSP is enabled, registration of storage backends are only allowed for admin. i.e registration of storage backends are not allowed for users or non-admin users. Because, the main use case of SSP is abstracting the storage backends from users.
+
+Click on (+) for registering a storage backend
+
+![multi-cloud backend image  ](register-backend-0.png?raw=true)
+
+and then provide valid region, endpoint, Access and Secret Key parameters
+
+![multi-cloud backend image  ](registerBackend.png?raw=true)
+
+### Create Storage Service Plan: [Admin Operation]
+
+**Note**: Again this is an admin operation and non-admin users are not allowed to create service plans. Non-admin users can only list and select the service plan during bucket creation.
+
+1. Once backends are registered, storage service plans can be created.
+
+  a.  Select the Storage Service Plan
+
+  ![SSP image  ](SSP.png?raw=true)
+
+  b. Create the Storage Service Plan
+
+  ![SSP Create image  ](SSP_Create.png?raw=true)
+
+### Create bucket and upload the object using SSP
+After creating new backend follow the steps given below to create a bucket and upload an object in that bucket:
+
+1. Launch Resource from SODA home page
+
+2. Create a new bucket by selecting appropriate SSP plan
+
```

Comment:
If a user is assigned to multiple tenants and these tenants are assigned to different Service plans, the user has to switch tenants to use that service plan. This can be added as another step.

Reply:
I think it's ok for now.